modules/gdscript/gdscript_editor.cpp

```diff
@@ -3178,6 +3178,13 @@ ::Error GDScriptLanguage::lookup_code(const String &p_code, const String &p_symb
 		return OK;
 	}
 
+	if ("assert" == p_symbol || "preload" == p_symbol) {
```

Comment:
These could be put as `||` checks in the above `function_exists()` one since the block is the same.

Reply:
Maybe also add a short comment that `assert` and `preload` are pseudo-functions (actually keywords).